packages/ocs/dashboards/persistent-internal/capacity-trend-card/capacity-trend-card.tsx

```diff
@@ -34,6 +45,34 @@ import { ODFSystemParams } from '../../../types';
 
 const parser = compose((val) => val?.[0]?.y, getInstantVectorStats);
 
+function calculateDaysUp(response: PrometheusResponse): number | null {
```

Comment:
```suggestion
const calculateDaysUp = (response: PrometheusResponse): number | null => {
```
Please use arrow functions